backend/routes.js

```diff
@@ -257,8 +266,40 @@ export default class Servers {
                         existingUsers.push({username, accessToken: access_token});
                         status = 201;
                     }
-                    saveSetting('USERS', existingUsers);
-                    res.json(status, {});
+
+                    if (contains(username, getSetting('ALLOWED_USERS'))) {
+                        res.setCookie('plotly-auth-token', access_token, {'path': getSetting('DB_CONNECTOR_URL')});
+
+                        const db_connector_access_token = generateAndSaveAccessToken();
```

Comment:
I just noticed that this the flow will _reset_ the access token. So, if 5 people log in at the same time, the access token will get reset after each person. 
I got around this in the dash-auth package by saving the expiry date of the access token and my `generateAndSaveAccessToken` looked at the date before generating:
```
generateAndSaveAccessToken() {
    if (getSetting('ACCESS_TOKEN_EXPIRATION') > now()) {
         const access_token = uuid4();
         saveSetting('ACCESS_TOKEN', access_token);
         saveSetting('ACCESS_TOKEN_EXPIRATION', now + 5 minutes);
         return access_token;
    } else {
        return getSetting('ACCESS_TOKEN');
    }
}
```

It's an uncommon case but maybe should tighten it up